.github/workflows/ci.yaml

```
@@ -105,14 +109,11 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust ${{ env.rust_clippy }}
```

hmm, this looks like it references an env var that doesn't exist, it needs to be:
```suggestion
      - name: Install Rust ${{ env.RUST_FMT_CLIPPY }}
```